XLS-47d-PriceOracles/README.md

```diff
@@ -0,0 +1,468 @@
+
+The price data to aggregate is selected based on specific criteria. The most recent Price Oracle object is obtained for the specified oracles. The most recent `LastUpdateTime` among all objects is chosen as the upper time threshold. A Price Oracle object is included in the aggregation dataset if it satisfies the conditions of containing the specified `base_asset`/`quote_asset` pair, including the `AssetPrice` field, and its `LastUpdateTime` is within the time range of (upper threshold - time threshold) to the upper threshold. If a Price Oracle object doesn't contain the `AssetPrice` for the specified token pair, then up to three previous Price Oracle objects are examined and include the first one that fulfills the criteria.
+
+The `get_aggregate_price` fails if:
+
+- The oracles array size is either 0 or greater than 200.
+- The oracles array's object doesn't include `account` or `oracle_sequence` or those fields have invalid value.
+- `base_asset` or `quote_asset` are missing.
+- `trim` or `time_threshold` contain invalid uint value.
+- If the resulting data set is empty.
+
+###### Example of get_aggregate_price API JSON request
+
+    {
+    "method": "get_aggregate_price",
+    "params": [
+        {
+            "ledger_index": "current",
+            "base_asset": "XRP",
+            "quote_asset": "USD",
+            "flags": 7,
+            "trim": 20,
+            "oracles": [
+              {
+                "account": "rp047ow9WcPmnNpVHMQV5A4BF6vaL9Abm6,
+                "oracle_sequence": 34
+              },
+              {
+                "account": "rp147ow9WcPmnNpVHMQV5A4BF6vaL9Abm7,
+                "oracle_sequence": 56
+              },
+              {
+                "account": "rp247ow9WcPmnNpVHMQV5A4BF6vaL9Abm8,
+                "oracle_sequence": 2
+              },
+              {
+                "account": "rp347ow9WcPmnNpVHMQV5A4BF6vaL9Abm9,
+                "oracle_sequence": 7
+              },
+              {
+                "account": "rp447ow9WcPmnNpVHMQV5A4BF6vaL9Abm0,
+                "oracle_sequence": 109
+              }
+            ]
+        }
+    ]
+    }
+
+##### Output fields
+
+On success, the response data contains the following fields:
+
+- `entire_set` is an object of the following fields:
+  - `size` is the size of the data set used to calculate the statistics.
+  - `average` is the simple average.
+  - `standard_deviation` is the standard deviation.
+- `trimmed_set` is an object, which is included in the response if `trim` fields is set. The object has the following fields:
+  - `size` is the size of the data set used to calculate the statistics.
+  - `average` is the simple average.
+  - `standard_deviation` is the standard deviation.
+- `median` is the median.
+- `time` is the most recent time stamp out of all `LastUpdateTime` values.
```

Comment:
if my proposal of using ISO8601 time, then this line does not need to change, but the examples below do.